pages/techgenius.md

```diff
@@ -46,13 +46,20 @@ Please open `git-bash` by pressing the Windows key to open the home screen, and
 
 Please open a terminal session.
 
-**Enter the following commands at the command prompt:**
+**Next, enter the following commands at the command prompt:**
 
 ```bash
 git clone https://github.com/dogi/ole--vagrant-community.git
 cd ole--vagrant-community
 vagrant up
 ```
+**NOTE**: At this point you will have two virtual machines installed and running on your computer. In order to conserve hardware resources and to keep things clean and simple or in case you encounter an error, it is best to halt the original (ole--vagrant-vi) community.
+```
+1. vagrant halt
+2. cd ..
+3. cd ole--vagrant-community
+4. vagrant up

```

Comment:
Since the code snippet above already include `cd ole--vagrant-community` and `vagrant up`, it doesn't make sense to include `vagrant up` again here. Would it be better to kind of reformat the code snippet in this note section?

Something like this?
```bash
# cd to ole--vagrant-vi directory
vagrant halt
```

Reply:
@cramartef Thanks for the update, but my main point was that [line 53-54](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1212/files?diff=split#diff-26b52773257dacc825932efccc01c45bR53) already ask people to do `cd ole--vagrant-community` and `vagrant up`, so having it on [line 61-62](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1212/files?diff=split#diff-26b52773257dacc825932efccc01c45bR61) again is not necessary.

Reply:
Ok. I did it.